pemFioi/quickpi/blocklyQuickPi_lib.js

```diff
@@ -1617,116 +1617,120 @@ var getContext = function (display, infos, curLevel) {
     // Some data can be made accessible by the library through the context object
     context.quickpi = {};
 
+    // we put that outside of the function #context.getConceptList because that is our only way to know the which sensor
```

Comment:
I don't understand why you took the list out ; you could just call `context.getConceptList();`.

Reply:
I done this to prevent calls to getConceptList because I just need the fields that I have put outside.